Dockerfile

```diff
@@ -8,14 +8,12 @@ RUN apt-get update && \
     pecl install xdebug && docker-php-ext-enable xdebug && \
     docker-php-ext-install -j$(nproc) pdo_mysql zip
 
-COPY --from=composer:2.5.8 /usr/bin/composer /usr/local/bin/composer
+COPY --from=composer:2.6.2 /usr/bin/composer /usr/local/bin/composer
+
+ENV COMPOSER_ALLOW_SUPERUSER=1
 
 WORKDIR /code
 
 COPY ./ ./
 
-ENV COMPOSER_ALLOW_SUPERUSER=1
-
-RUN composer install
-
-CMD ["./vendor/bin/phpunit", "--testdox"]
+CMD ["bash", "-c", "composer install && ./vendor/bin/phpunit --testdox"]
```

Comment:
Out of curiosity, what's the advantage of moving `composer install` into `CMD` from its separate `RUN`? Does it mean that the built image will be without composer dependencies and they will be installed before running tests?

Reply:
The working directory is mounted into the container by docker-compose, so dependencies installed the docker image are ignored.
Also, if you ran composer install with PHP 8.2 locally and try to run phpunit in the container with PHP 8.1, it will fail. That's also why I add `config.platform.php: 8.1` in composer.json

Reply:
I see, thanks for explaining!